### ISSUE TITLE
Missing aria-orientation label for horizontal min pointer

### ISSUE THREAD

**Issue by memamdie** [angular-slider/ngx-slider#163]
https://github.com/angular-slider/ng5-slider/blob/master/src/ng5-slider/lib/slider.component.ts#L989

The min pointer aria orientation is only set when the orientation is vertical. suggested fix is

```
this.minHandleElement.ariaOrientation = this.viewOptions.vertical ? 'vertical' : 'horizontal';
```

**Comment by piotrdz** [comment]
Thanks, I pushed the change as suggested to master, awaiting next release.
